config/mycnf/master_mysql80.cnf

```diff
@@ -0,0 +1,34 @@
+# Options for enabling GTID
+# https://dev.mysql.com/doc/refman/5.6/en/replication-gtids-howto.html
+gtid_mode = ON
+log_bin
+log_slave_updates
+enforce_gtid_consistency
+
+# Crash-safe replication settings.
+master_info_repository = TABLE
+relay_log_info_repository = TABLE
+relay_log_purge = 1
+relay_log_recovery = 1
+
+# Native AIO tends to run into aio-max-nr limit during test startup.
+innodb_use_native_aio = 0
+
+# Semi-sync replication is required for automated unplanned failover
+# (when the master goes away). Here we just load the plugin so it's
+# available if desired, but it's disabled at startup.
+#
+# If the -enable_semi_sync flag is used, VTTablet will enable semi-sync
+# at the proper time when replication is set up, or when masters are
+# promoted or demoted.
+plugin-load = rpl_semi_sync_master=semisync_master.so;rpl_semi_sync_slave=semisync_slave.so
```

Comment:
is this something we can delete for 8.0?

Reply:
according to the manual you still need the plugin in mysql 8.0 

https://dev.mysql.com/doc/refman/8.0/en/replication-semisync-installation.html